### ISSUE TITLE
Disabling Developer Preview does not disable it

### ISSUE THREAD

**Issue by jeserkin** [MicrosoftDocs/msteams-docs#8776]
Disabling Developer Preview does not disable it

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 008da4e2-979e-c396-db28-4acf57deb530
* Version Independent ID: f1413f77-5eb3-e1fd-9235-d322d6e4eb6c
* Content: [Public developer preview for Microsoft Teams - Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/resources/dev-preview/developer-preview-intro#disable-developer-preview)
* Content Source: [msteams-platform/resources/dev-preview/developer-preview-intro.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/resources/dev-preview/developer-preview-intro.md)
* Service: **msteams**
* GitHub Login: @surbhigupta
* Microsoft Alias: **lajanuar**

**Comment by Meghana-MSFT** [comment]
Thank you for reporting this issue, could you please let us know if you are facing this issue in Windows or Mac machine? Also please share your Teams version.

**Comment by Jocelynncanzoneri** [comment]
Yes I'm having trouble taking to any kind of payout can anyone help me I'd
love to get paid for something it seems like the better is or betting and
if they go with whatever I do for the day they win if not they don't but
I'm still not getting anywhere I'm getting anything. Thank you so might
help me find out how I click on that account

**Comment by jeserkin** [comment]
> Thank you for reporting this issue, could you please let us know if you are facing this issue in Windows or Mac machine? Also please share your Teams version.

* I am having issue on Windows machine.
* You have Microsoft Teams Version 1.6.00.12455 (64-bit). It was last updated on 01-Jun-23.

**Comment by Meghana-MSFT** [comment]
@jeserkin - We are also able to repro this issue, we have raised a bug for the same. We will keep you posted on the updates. Thank you.

**Comment by jeserkin** [comment]
Any news regarding this issue? I see it is still present. Very frustrating. I am used to `Quit` being last option in context menu for MS Teams system tray icon, at the moment as mentioned it is `Open DevTools`.

**Comment by Meghana-MSFT** [comment]
@jeserkin - We currently do not have any update regarding this issue. We are following up with engineering team. Please be assured that we will keep you posted on the updates. Thank you.

**Comment by jeserkin** [comment]
I wasn't planning on asking again, but since month has passed and there is no change, I wonder. Does it bother only me or is there some workaround, that people use? I assume MS Teams is widely used these days.

**Comment by Meghana-MSFT** [comment]
@jeserkin - Apologies for the inconvenience. Team is looking into this issue. We will keep you posted on the updates. Thank you.

**Comment by jeserkin** [comment]
I give up. I feel like this will never change.
